### PR TITLE
Universal output for velocity calib

### PR DESCRIPTION
**Pull request by wictus**
Make output from velocity calibration the same as for UniversalCalib (same as in TimeCalibration)

### CODE REVIEW

File: VelocityCalibration/DeltaTFinder.cpp [hunk 1]
```diff
@@ -75,6 +75,11 @@ bool DeltaTFinder::init()
     pos += "_std::string";
     if (isOptionSet(fParams.getOptions(), pos)) {
       auto res = retrievePositionAndFileName(getOptionAsString(fParams.getOptions(), pos));
+      if ( file_path.find_last_of("/") ) 
+	file_path = file_path.substr( file_path.find_last_of("/")+1,  file_path.find(".") - file_path.find_last_of("/")-1 );
```
Comment (wkrzemien) [inline VelocityCalibration/DeltaTFinder.cpp]: please add braces or at least one tab

Comment (wkrzemien) [inline VelocityCalibration/DeltaTFinder.cpp]: I am not 100% sure what are you doing here, but check JPetCommonTools maybe there is a method that can be used here, Also, boost:filesystem has a set of methods to deal with pahts

File: VelocityCalibration/estimateVelocity.cpp [hunk 2]
```diff
@@ -159,6 +172,27 @@ int findScintillatorIn( std::vector<scintData>& data, int ID)
 	return -1;
 }
 
+bool ifThresholdIsIn( scintData& data, char label)
+{
```
Comment (wkrzemien) [inline VelocityCalibration/estimateVelocity.cpp]: consider something like
return std::any_of(data.cbegin(), data.cend(), [label]( const TYPE_OF_THRESHOLD& thr) {return label == threshold.thresholdLabel; });

File: VelocityCalibration/DeltaTFinder.cpp (hunk 1 above)
Comment (wictus) [reply VelocityCalibration/DeltaTFinder.cpp]: I needed filename, without path and extensions. I used JPetCommonTools::extractFileNameFromFullPath as you suggested.

File: VelocityCalibration/estimateVelocity.cpp [hunk 3]
```diff
@@ -87,37 +87,50 @@ void parserTest(std::vector<scintData>& data, const char thrLabel)
...
-					positionsForFit.push_back( position.positionValue );
-					timesForFit.push_back( position.deltaT );
-					errorsForFit.push_back( position.deltaTError);
-					std::cout << positionsForFit.size() << std::endl;
-					std::cout << timesForFit.size() << std::endl;
-				}
+			std::vector<double> positionsForFit;
+			std::vector<double> timesForFit;	
+			std::vector<double> errorsForFit;
+			for( auto position : threshold.positions )
```
[the remark this answers is not in the document]
Comment (wictus) [reply VelocityCalibration/estimateVelocity.cpp]: Changed, I only go through those structures and extract what I need.

**Comment by wkrzemien** [comment]
retest this please

**Comment by wictus** [comment]
I did one small fix, now it works with data properly.
Thanks for a very nice improvement in estimateVelocity.cpp

**Comment by grey277** [comment]
run velocity tests

**Comment by wkrzemien** [comment]
Why those tests are not passing? @wictus cause of the new reference files needed?

**Comment by grey277** [comment]
run velocity tests

**Comment by grey277** [comment]
retest this please

**Comment by wkrzemien** [comment]
If I understand correctly @kkacprzak  there is an issue with user params. Can you fix it?

**Comment by wictus** [comment]
Userparams.json was removed from repo

**Pull request merged**